docs/admin-network-policy.md

```diff
@@ -0,0 +1,118 @@
+# AdminNetworkPolicy API Support in Antrea
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Usage](#usage)
+  - [Sample specs for AdminNetworkPolicy and BaselineAdminNetworkPolicy](#sample-specs-for-adminnetworkpolicy-and-baselineadminnetworkpolicy)
+  - [Relationship with Antrea-native Policies](#relationship-with-antrea-native-policies)
+<!-- /toc -->
+
+## Introduction
+
+Kubernetes provides the NetworkPolicy API as a simple way for developers to control traffic flow to their applications.
+While NetworkPolicy is embraced throughout the community, it was designed for developers instead of cluster admins.
+Therefore, traits such as the lack of explicit deny rules make securing workloads at the cluster level difficult.
+The Network Policy API working group (subproject of Kubernetes SIG-Network) has then introduced the
+[AdminNetworkPolicy APIs](https://network-policy-api.sigs.k8s.io/api-overview/) which aims to solve the cluster admin
+policy usecases.
+
+Starting with v1.13, Antrea supports the `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` API types, except for
+advanced namespace selection mechanisms (namely `sameLabels` and `notSameLabels` rules) which are still in experimental
+phase and not required as part of conformance.
+
+## Prerequisites
+
+AdminNetworkPolicy was introduced in v1.13 as an alpha feature and is disabled by default. A feature gate,
+`AdminNetworkPolicy`, must be enabled in antrea-controller.conf in the `antrea-config` ConfigMap when Antrea is deployed:
```

Comment:
Since there is a new feature gate added, I think you should also update the doc `feature-gates.md`.
I suppose this feature depends on `AntreaPolicy` feature? maybe add a few words about the dependency.

Reply:
`feature-gates.md` in updated in the PR of AdminNP implementation. Updated the `AntreaPolicy` featureGate dependency.